src/components/Profile/Profile.js

```diff
@@ -50,9 +51,42 @@ class UserProfile extends Component {
       }));
     }
   }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    console.log("submitt");
+    if ( this.state.newpassword!=="" && this.state.confnewpassword==="" ) {
+      alert("Please confirm your password !!!");
+      return;
+    }
+    if ( this.state.newpassword==="" && this.state.confnewpassword!=="" ) {
+      alert("Please enter both password fields !!!");
+      return;
+    }
+    if ( this.state.newpassword!==this.state.confnewpassword ) {
+      alert("Password do not match !!!");
+      return;
+    }
+    const update={};
+    const body={};
+    if ( this.state.name!=="" ) {
+      body.name=this.state.name;
+    }
+    if ( this.state.email!=="" ) {
+      body.email=this.state.email;
+    }
+    if ( this.state.newpassword!=="" ) {
+      body.password=this.state.newpassword;
+    }
+    update.updatedUser=body;
+    update.userId=this.props.authUser.id;
+    console.log(update);
```

Comment:
Assign update object as `const update = { updatedUser:body,userId:this.props.authUser.id}`
Remove console.log